resources/google_drive_upload/refresh.sh

```diff
@@ -0,0 +1,12 @@
+#!/bin/bash
+
+set -e
+
+prefix="resources/google_drive_upload"
+
+folder="notebooks"
+script="$prefix/drive.py"
+folderId="`cat $prefix/folderId.txt`"
+
```

Comment:
Let's inline it all to the main line or extract the json to a variable as well for consistency.

Reply:
Done.